test/integration-test.py

```diff
@@ -265,6 +274,7 @@ def main():
             die(ExitStatus.Error)
         run_node_test()
         run_caa_node_test()
+        run_all_chall_tests()
```

Comment:
Due to #1280 we can't run `run_node_test` multiple times in a row because it will fail. When that is fixed this should probably be changed so that `run_node_test` runs for each implemented challenge instead of having separate commands for the main test, CAA test, and this test.